screens/Settings/index.tsx

```diff
@@ -33,7 +40,10 @@ const Settings = () => {
   return (
     <>
       <List.Item title="Clear Data" onPress={clearData} />
-      <List.Item title="Privacy Policy" onPress={opnePrivacyPolicy} />
+      <Divider />
+      <List.Item title="Privacy Policy" onPress={openPrivacyPolicy} />
+      <Divider />
+      <List.Item title="About" onPress={() => navigation.navigate('AboutScreen')} />
```

Comment:
I quite like the look of a divider separating each ListItem? Happy to revert it though.

![Simulator Screen Shot - iPhone 8 - 2022-02-06 at 16 56 20](https://user-images.githubusercontent.com/6009338/152691816-0a392244-6dce-40e1-8fb1-72bdf74c282a.png)